www-apps/radicale/radicale-2.1.8.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python{3_4,3_5,3_6} )
+
+inherit eutils distutils-r1 user
```

Comment:
Please sort lexically whenever possible.

Reply:
Fixed in 4d6e56e141f0.